.github/workflows/build.yaml

```diff
@@ -0,0 +1,65 @@
+# name of the work flow
+name: Build
+on: [pull_request]
+jobs:
+  build:
+    if: ${{ !contains(github.event.head_commit.message, 'ci skip') }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [macos-latest]
```

Comment:
Do we really need mac-os? Surely we can run this build on ubuntu.

Reply:
Yes, Will, we can do it on Ubuntu and windows as well.